ms/AuthServices.java

```diff
@@ -0,0 +1,185 @@
+            
+            String sql = "INSERT INTO users(user_name, password) VALUES (\""+credentials.getUsername()+"\",\""+credentials.getPassword()+"\")";
+
+            // execute the update
+
+            stmt.executeUpdate(sql);
+
+            // clean up the environment
+
+            stmt.close();
+            conn.close();
+            stmt.close(); 
+            conn.close();
+
+        } catch(Exception e) {
+            ReturnString = e.toString();
+        } 
+        
+        return(ReturnString);
+
+    }
+
+    public String AuthenticateUser(UserCredentials credentials) throws RemoteException 
+    {
+        // Local declarations
+        Connection conn = null;		// connection to the orderinfo database
+        Statement stmt = null;		// A Statement object is an interface that represents a SQL statement.
+        Boolean authenticated = false;	// Return boolean. True when authenticated
+        String token = null;
+
+        try
+        {
+            // Here we load and initialize the JDBC connector. Essentially a static class
+            // that is used to provide access to the database from inside this class.
+
+            Class.forName(JDBC_CONNECTOR);
+
+            //Open the connection to the orderinfo database
+
+            //System.out.println("Connecting to database...");
+            conn = DriverManager.getConnection(DB_URL,USER,PASS);
+
+            // Here we create the queery Execute a query. Not that the Statement class is part
+            // of the Java.rmi.* package that enables you to submit SQL queries to the database
+            // that we are connected to (via JDBC in this case).
+
+            // System.out.println("Creating statement...");
+            stmt = conn.createStatement();
+            
+            String sql;
+            sql = "SELECT * FROM users where user_name='" + credentials.getUsername()+"'";
+            ResultSet rs = stmt.executeQuery(sql);
+
+            // Extract data from result set. Note there should only be one for this method.
+            // I used a while loop should there every be a case where there might be multiple
+            // orders for a single ID.
+
+            while(rs.next())
+            {
+                //Retrieve by column name
+                int id  = rs.getInt("user_id");
+                String username = rs.getString("user_name");
+                String password = rs.getString("password");
+
+                if (password.equals(credentials.getPassword()))
+                {
+                    authenticated = true;
+                    token = CreateToken(credentials);
+                    sessions.put(token, credentials);
+                }
+            }
+
+            //Clean-up environment
+
+            rs.close();
+            stmt.close();
+            conn.close();
+            stmt.close(); 
+            conn.close();
+
+        } catch(Exception e) {
+            System.out.println("AuthServices err: " + e.getMessage());
```

Comment:
Do we need to use the logging feature going forward instead of System.out.println? If so, we can mark this as TODO